modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -107,6 +107,14 @@ CV__DNN_INLINE_NS_BEGIN
         DNN_TARGET_CPU_FP16, // Only the ARM platform is supported. Low precision computing, accelerate model inference.
     };
 
+    // Shared reduction enum for DNN loss layers
+    enum LossReduction
+    {
+        DNN_LOSS_REDUCTION_NONE = 0,
```

Comment:
for some enumerations we use DNN_ prefix, some some we don't. Can we normalize it?